src/wp-includes/blocks.php

```diff
@@ -751,6 +751,9 @@ function get_hooked_blocks( $name ) {
 	$block_types = WP_Block_Type_Registry::get_instance()->get_all_registered();
 	$hooked_blocks = array();
 	foreach ( $block_types as $block_type ) {
+		if ( ! property_exists( $block_type, 'block_hooks' ) ) {
```

Comment:
> I wonder if/why the block_hooks field isn't available at all (rather than defaulting to the empty array)?

It might be the tests for REST API that try to inject invalid values. `! empty( $block_type->block_hooks )` check would be faster and it would also cover the case when there is an empty array.

Reply:
It might be an issue with the missing test clean up. I'm trying unregistering the block with invalid data: https://github.com/WordPress/wordpress-develop/pull/5261/commits/d78370204077268a39ed5ffc30f053abcf0dbb48.